recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "23e3af619a4859008745ac61a6040a230023cd4e5c4c667d9b8ea248726f47ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
```

Comment:
There doesn't seem to be any real code in https://github.com/tdegeus/pdfcombine?

Reply:
There is one Python-script (executable) that allows one concatenate PDFs (in a single command).

Reply:
Also note that I deliberately split an old repository for sharing only package manager worthy code.

Reply:
Note that based on your remark I also updated the readme of the the Github repository